keyboards/divergetm2/keymaps/xtonhasvim/rules.mk

```diff
@@ -0,0 +1,7 @@
+ifndef QUANTUM_DIR
```

Comment:
Lines 1-3 aren't needed; they're a holdover from a previous `make` system that is now deprecated.

Reply:
Yes, please remove this ifndef block.